### ISSUE TITLE
refactor-nrepl and clojure 1.9.0-beta1

### ISSUE THREAD

**Issue by slipset** [clojure-emacs/refactor-nrepl#205]
### clj-refactor.el and refactor-nrepl version information

clj-refactor 2.4.0-SNAPSHOT (package: 20170720.712), refactor-nrepl 2.4.0-SNAPSHOT

### CIDER version information
```
;; CIDER 0.16.0snapshot (package: 20171001.112), nREPL 0.2.12
;; Clojure 1.9.0-beta1, Java 1.8.0_144
```

With Clojure 1.9.0-beta1, running M-x cider-jack-in gives the following warnings:
```
WARNING: boolean? already refers to: #'clojure.core/boolean? in namespace: mranderson047.toolsanalyzerjvm.v0v6v9.toolsanalyzer.v0v6v7.clojure.tools.analyzer.utils, being replaced by: #'mranderson047.toolsanalyzerjvm.v0v6v9.toolsanalyzer.v0v6v7.clojure.tools.analyzer.utils/boolean?
WARNING: boolean? already refers to: #'clojure.core/boolean? in namespace: mranderson047.toolsanalyzerjvm.v0v6v9.toolsanalyzer.v0v6v7.clojure.tools.analyzer, being replaced by: #'WARNING: boolean? already refers to: #'clojure.core/boolean? in namespace: mranderson047.toolsanalyzerjvm.v0v6v9.toolsanalyzer.v0v6v7.clojure.tools.analyzer.utils, being replaced by: #'mranderson047.toolsanalyzerjvm.v0v6v9.toolsanalyzer.v0v6v7.clojure.tools.analyzer.utils/boolean?
WARNING: boolean? already refers to: #'clojure.core/boolean? in namespace: mranderson047.toolsanalyzerjvm.v0v6v9.toolsanalyzer.v0v6v7.clojure.tools.analyzer, being replaced by: #'mranderson047.toolsanalyzerjvm.v0v6v9.toolsanalyzer.v0v6v7.clojure.tools.analyzer.utils/boolean??
```

It seems that the ns `mranderson047.toolsanalyzerjvm.v0v6v9.toolsanalyzer.v0v6v7.clojure.tools.analyzer.utils` comes from refactor-nrepl.

**Comment by benedekfazekas** [comment]
well, really that comes from tools.analyzer.jvm only it is inlined. we also get this warning on the travis build with an earlier alpha but the build passes ok.

**Comment by benedekfazekas** [comment]
why do you think it is a problem?

**Comment by slipset** [comment]
I know it works as it should, but I don't think it should emit warnings.
FWIW, tools.analyzer fixed this in 
https://github.com/clojure/tools.analyzer/commit/c375596ca76f79d701132d75c4021aeb4a3d20d4#diff-f9be9d3df305891a7d178b5105b25f10, which I believe is available in version 0.6.9, and from the warnings above, it seems like something is using tools analyzer 0.6.7.

So my initial thought would be that the deps to tools.analyzer needs to be updated to 0.6.9

**Comment by benedekfazekas** [comment]
good shout. thanks for flagging this!

**Comment by slipset** [comment]
My pleasure :). Sorry I'm not able to submit a PR to fix this, but I can't seem to grok how the deps are inlined and stuff here.

**Comment by benedekfazekas** [comment]
no worries. cljr work is piling up so I will come to this deps update too eventually. ta!

**Comment by expez** [comment]
Fixed in latest snapshot.

**Comment by daveyarwood** [comment]
I was having this issue too on refactor-nrepl 2.3.1. Can confirm it's fixed in 2.4.0-20171124.

Thanks for the fix!
